docs/app/css/style.css

```diff
@@ -185,10 +185,6 @@ pre {
 pre > code.highlight {
   padding: 16px;
   font-weight: 400;
-  -webkit-user-select: initial;
-  -moz-user-select: initial;
-  -ms-user-select: initial;
```

Comment:
Yeah, I don't understand why says MDN `none`. Perhaps Firefox set it to `none` for certain elements. css-tricks.com says the default is `auto`. `auto` option which would probably make more sense than `initial` if we decided to keep the code.

There must have been some CSS that disabled user-select but I can't find anything in master. I don't it's relevant anymore and it looks like the block can be removed safely.